php-console.js

```diff
@@ -181,6 +181,54 @@
                 e.preventDefault();
             });
         }
+        
+        //Save
+        if (window.localStorage) {
+            $('.statusbar .save').on('click', function (e) {
+            	var textToWrite = editor.getSession().getValue();
+            	var textFileAsBlob = new Blob([textToWrite], {type:'text/plain'});
+            	var fileNameToSaveAs = document.getElementById("inputFileNameToSaveAs").value;
+
+            	var downloadLink = document.createElement("a");
+            	downloadLink.download = fileNameToSaveAs;
+            	downloadLink.innerHTML = "Download File";
+            	if (window.webkitURL != null)
+            	{
+            		// Chrome allows the link to be clicked
+            		// without actually adding it to the DOM.
+            		downloadLink.href = window.webkitURL.createObjectURL(textFileAsBlob);
+            	}
+            	else
+            	{
+            		// Firefox requires the link to be added to the DOM
```

Comment:
Dunno. Copied it from here:
https://thiscouldbebetter.wordpress.com/2012/12/18/loading-editing-and-saving-a-text-file-in-html5-using-javascrip/